src/Console/Database/FixHtmlEncodingCommand.php

```diff
@@ -0,0 +1,208 @@
+ *
+ * ---------------------------------------------------------------------
+ *
+ * LICENSE
+ *
+ * This file is part of GLPI.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ *
+ * ---------------------------------------------------------------------
+ */
+
+namespace Glpi\Console\Database;
+
+use CommonDBTM;
+use Glpi\Console\AbstractCommand;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+
+class FixHtmlEncodingCommand extends AbstractCommand
+{
+    /**
+     * Error code returned when a specified iteltype does not exists
+     *
+     * @var integer
+     */
+    const ERROR_ITEMTYPE_NOT_FOUND = 1;
+
+    /**
+     * Error code returned when at least one item id is not found
+     *
+     * @var integer
+     */
+    const ERROR_ITEM_ID_NOT_FOUND = 2;
+
+    /**
+     * Error code returned when at least one field is not found
+     *
+     * @var integer
+     */
+    const ERROR_FIELD_NOT_FOUND = 3;
+
+    /**
+     * Error code returned when update of an item failed
+     *
+     * @var integer
+     */
+    const ERROR_UPDATE_FAILED = 4;
+
+    protected function configure()
+    {
+        parent::configure();
+
+        $this->setName('glpi:database:fix_html_encoding');
+        $this->setAliases(['db:fix_html']);
+        $this->setDescription(__('Enable timezones usage.'));
+
+        $this->addOption(
+            'itemtype',
+            null,
+            InputOption::VALUE_REQUIRED,
+            __('Itemtype to fix')
+            . PHP_EOL
+            . __('"--itemtype ITILFollowup" will fix an item of type ITILFollowup')
+            . PHP_EOL
+        );
+
+        $this->addOption(
+            'id',
+            null,
+            InputOption::VALUE_IS_ARRAY | InputOption::VALUE_REQUIRED,
+            __('Id of the item to fix')
+            . PHP_EOL
+            . __('"--id 42 will fix the item with id 42')
+            . PHP_EOL
+        );
+
+        $this->addOption(
+            'field',
+            null,
+            InputOption::VALUE_IS_ARRAY | InputOption::VALUE_REQUIRED,
+            __('Field of the item to fix')
+            . PHP_EOL
+            . __('"--field content" will fix the field "content"')
+            . PHP_EOL
+        );
+
+        $this->addUsage('--field name --field content');
```

Comment:
Usage example should contains all required options. e.g. `--itemtype=ITILFollowup --id=42 --field=content`.